_pages/license.md

```diff
@@ -0,0 +1,68 @@
+---
+title: License
+permalink: /license/
+sidebar:
+  nav: legal
+---
+The contents of this website are published under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/) under the
+Copyright © 2020 of the SORSE20 committee.
+
```

Comment:
Any reason to not have the LICENSE as an include (just plop in the includes folder) and then just include it here? if there are changes, we'd want both the rendered and "hard coded" version in the site to be synced.

Reply:
Hmm wait this is listing all the minimal mistakes licenses. Is that preferable to just linking to the minimal mistakes repository, with a summary sentence? We would again need to keep these updated.